client/src/components/game/gui/panels/chat/ChatLine.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./ChatLine.scss";
+
+function ChatLine({
+    scope, displayName, message, getScopeColor,
+}) {
+    return (
+        <p className={`chat-line ${getScopeColor(scope)}`}>
+            {`${displayName}: ${message}`}
```

Comment:
Still could do with making the display name bold so it is a bit more distinct from the message.
```js
<p className={`chat-line ${getScopeColor(scope)}`}>
    <span className="name">{`${displayName}: `}</span>
    <span>{message}</span>
</p>
```
![image](https://user-images.githubusercontent.com/9785268/114310220-fdb8d500-9ae1-11eb-9698-1ae55c21d398.png)